src/features/Sync/SyncSlice.ts

```diff
@@ -454,14 +580,28 @@ function* fetchAll(): SagaIterator {
 }
 
 function* root(): SagaIterator {
-    yield takeEvery(SyncAction.updateHost, updateHost);
-    yield takeEvery(SyncAction.updatePort, updatePort);
+    yield takeEvery(SyncAction.updateLoginUrl, updateLoginUrl);
+    yield takeEvery(SyncAction.updateSensorUrl, updateSensorUrl);
+    yield takeEvery(SyncAction.updateTemperatureLogUrl, updateTemperatureLogUrl);
+    yield takeEvery(SyncAction.updateTemperatureBreachUrl, updateTemperatureBreachUrl);
     yield takeEvery(SyncAction.updateUsername, updateUsername);
     yield takeEvery(SyncAction.updatePassword, updatePassword);
+    yield takeEvery(SyncAction.authenticate, authenticate)
+    yield takeEvery(SyncAction.authenticateSuccess, authenticateSuccess);
+    yield takeEvery(SyncAction.authenticateFailure, authenticateFailure);
+    yield takeEvery(SyncAction.syncSensors, syncSensors);
+    yield takeEvery(SyncAction.syncSensorsSuccess, syncSensorsSuccess);
+    yield takeEvery(SyncAction.syncSensorsFailure, syncSensorsFailure);
+    yield takeEvery(SyncAction.syncTemperatureLogs, syncTemperatureLogs);
+    yield takeEvery(SyncAction.syncTemperatureLogsSuccess, syncTemperatureLogsSuccess);
+    yield takeEvery(SyncAction.syncTemperatureLogsFailure, syncTemperatureLogsFailure);
+    yield takeEvery(SyncAction.syncTemperatureBreaches, syncTemperatureBreaches);
+    yield takeEvery(SyncAction.syncTemperatureBreachesSuccess, syncTemperatureBreachesSuccess);
+    yield takeEvery(SyncAction.syncTemperatureBreachesFailure, syncTemperatureBreachesFailure);
```

Comment:
What are the plans for the failure sagas?

We might also want to `takeLatest` of a few of these

Reply:
Issued created (#92).